### PR TITLE
Add fields and update layout for Address Object

### PR DESCRIPTION
**Pull request by searheart**
# Critical Changes

# Changes
- Updated Address Page Layout to include Address Verification Section for future use

# Issues Closed

# Community Ideas Delivered

- [Address Verification Override](https://powerofus.force.com/s/idea/a1W80000006hxTHEAY/address-verification-override)

# New Metadata

New Fields
- Address__c.Exclude_from_Updates__c
- Address__c.Verified_Date__c
- Address__c.NCOA_Evaluation_Date__c
- Address__c.NCOA_Result_Code__c

# Deleted Metadata

### CODE REVIEW

**Comment by mpusto** [comment]
Can we make sure that this Community Idea is marked as delivered? https://powerofus.force.com/s/idea/a1W80000006hxTHEAY/address-verification-override

**Comment by force2b** [comment]
Great catch @mpusto. I'm going to create a WI for this as well so it's linked to the actual build in Wilma.

**Comment by force2b** [comment]
**lurch: attach W-039967

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001HtDdQAK" target="_blank">W-039967</a>

File: src/objects/Address__c.object [hunk 1]
```diff
@@ -166,6 +166,17 @@
         <trackTrending>false</trackTrending>
         <type>Checkbox</type>
     </fields>
+    <fields>
```
Comment (force2b) [inline src/objects/Address__c.object]: @sam-knox Would you mind reviewing these new fields added by the Insights team to the Address object? I believe Lauren Beller has looked at these as well.

**Comment by searheart** [comment]
@sam-knox changes requested have been made.

**Comment by force2b** [comment]
@sam-knox Would you mind reviewing the PR comment block for this PR? It makes a note about the page layout being changed, and though that is true, it's not something that anyone will ever see because we cannot deploy page layout changes to customers. I think what might be better to mention is that there are new fields added to the Address object for future use.

**Comment by sam-knox** [comment]
@force2b - New customers would get this field on their page layout wouldn't they?

**Comment by force2b** [comment]
@sam-knox No. The layout in the package is overridden by a layout in the trial org configuration.

**Comment by sam-knox** [comment]
@force2b - Ok, thanks. I updated the Changes section to focus on the addition of new fields.

**Pull request merged**